### ISSUE TITLE
Could you verify the implementation? Acc = 11% after domain adaptation

### ISSUE THREAD

**Issue by lindagaw** [corenel/pytorch-adda#27]
=== Evaluating classifier for encoded target domain ===
>>> only source <<<
Avg Loss = 14.961788177490234, Avg Accuracy = 56.140000%
>>> source and target <<<
Avg Loss = 8366.6220703125, Avg Accuracy = 11.350000%

I got accuracy = 11 after domain adaptation.

**Comment by zzzpc** [comment]
I set  d_learning_rate = 1e-3 and c_learning_rate = 1e-5 can get a good result. 
>>> source only <<<
Avg Loss = 1980.854672080592, Avg Accuracy = 75.000000%
>>> domain adaption <<<
Avg Loss = 97.09941973184284, Avg Accuracy = 92.903227%

**Comment by Hcshenziyang** [comment]
I got accuracy = 8 after domain adaptation……
![image](https://user-images.githubusercontent.com/94844905/143671574-ebd972bb-ca9c-489e-95d3-6ba5aab89f4e.png)

**Comment by ghost** [comment]
Did you solved this issue ? I'am also facing low DA accurcay

**Comment by yuhui-zh15** [comment]
See #29

**Comment by mashaan14** [comment]
I think what causes the low adaptation accuracy is that the class labels are swapped by the target encoder. This makes sense because it is an unsupervised task and the target encoder didn't see the class labels.

I've used this code on 2D data:
https://github.com/mashaan14/ADDA-toy

You can see in the attached image that the target encoder separates the classes well. But the class labels were swapped.

![Testing target data using target encoder](https://user-images.githubusercontent.com/61033787/219020643-959e7b16-65db-47d5-97e4-c2a7c733d858.png)
